Website/openrails.org/discover/version-1-5-1/index.php

```diff
@@ -0,0 +1,90 @@
+<?php include "../../shared/head_notrack.php" ?>
+    <link rel="stylesheet" href="../../shared/iframe/iframe.css" type="text/css" />
+  </head>
+
+  <body>
+    <div class="container"><!-- Centres content and sets fixed width to suit device -->
+<?php include "../../shared/banners/choose_banner.php" ?>
+<?php include "../../shared/banners/show_banner.php" ?>
+<?php include "../../shared/menu.php" ?>
+      <div class="row">
+        <div class="col-md-12">
+          <h1>Discover > Version 1.5.1</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-1"></div>
+        <div class="col-md-10">
+          <h2>New in Open Rails 1.5.1 (since 1.4)</h2>
+          <p>
+            Note: Version 1.5.1 is the same as Version 1.5 but fixes an extra 3 bugs.
```

Comment:
Move this note to discover 1.5 and remove discover 1.5.1 - program links to discover 1.5 page even for 1.5.1